pkg/sdk/identifier_helpers.go

```diff
@@ -14,6 +15,31 @@ type ObjectIdentifier interface {
 	FullyQualifiedName() string
 }
 
+func ParseObjectIdentifier(identifier string) (ObjectIdentifier, error) {
```

Comment:
Please add a comment also to this method (that this is temporary, and will be tested, improved, and used more widely during the identifiers rework)

Reply:
Also please in a comment mark that this is a copy of existing method in the provider helpers